Cargo.toml

```diff
@@ -124,8 +124,9 @@ log_wrappers = { path = "components/log_wrappers" }
 [dependencies.murmur3]
 git = "https://github.com/pingcap/murmur3.git"
 
-[dependencies.rocksdb_do_not_use]
+[dependencies.engine_rocksdb]
 git = "https://github.com/pingcap/rust-rocksdb.git"
+rev = "5f49f9ef202d8ad9515de2bb05c8395c1c66b8aa"
```

Comment:
Thanks for adding the rev.

Reply:
I think this will import some resistance for upgrade RocksDB.

Reply:
Why?

Reply:
I think what he means is in the past, `cargo update -p rocksdb` will do all the work, but now either removing the line before update or looking up the hash manually.

I personally prefer to remove the revision too as that's the whole point of keeping a .lock file.